packages/scene-composer/src/components/panels/scene-components/AnimationComponentEditor.tsx

```diff
@@ -0,0 +1,97 @@
+import React, { useCallback, useEffect, useMemo, useState } from 'react';
+import { FormField, Button, SpaceBetween, Autosuggest } from '@awsui/components-react';
+import { useIntl } from 'react-intl';
+
+import { IAnimationComponentInternal, ISceneNodeInternal } from '../../../store';
+import { getAnimationObjectByName } from '../../three-fiber/AnimationComponent/AnimationComponent';
+
+export type AnimationComponentEditorProps = {
+  node: ISceneNodeInternal;
+  component: IAnimationComponentInternal;
+  onUpdate;
+};
+export const AnimationComponentEditor: React.FC<AnimationComponentEditorProps> = ({
+  node,
+  component: animationComponent,
+  onUpdate,
+}: AnimationComponentEditorProps) => {
+  const { formatMessage } = useIntl();
+
+  const [panelnumber, setPanelNumber] = React.useState(0);
+  const animationList = getAnimationObjectByName(animationComponent.uri);
+  console.log(animationList)
+  const [selectedAnimations, setSelectedAnimations] = React.useState<string[]>();
+
+  const animationOptions = useMemo(
+    () =>
+      animationList?.animations.map((animation) => ({
+        value: animation.name,
+      })),
+    [animationList, node],
```

Comment:
This probably doesn't need to trigger on node. if node changes the Animationlist should change.